install/aws/dev/docker/php-fpm/etc/nginx/sites-available/wikijump

```diff
@@ -19,42 +19,39 @@
 # Default server configuration
 #
 server {
+        server_name wikijump;
         listen 80;
         listen [::]:80;
 
-        # SSL configuration
-        #
-        #listen 443 ssl;
-        #listen [::]:443 ssl;
-        keepalive_timeout   70;
-
-        #ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
-        #ssl_ciphers         AES128-SHA:AES256-SHA:RC4-SHA:DES-CBC3-SHA:RC4-MD5;
-        #ssl_certificate     /usr/local/nginx/conf/cert.pem;
-        #ssl_certificate_key /usr/local/nginx/conf/cert.key;
-        #ssl_session_cache   shared:SSL:10m;
-        #ssl_session_timeout 10m;
-        #
-        # Note: You should disable gzip for SSL traffic.
-        # See: https://bugs.debian.org/773332
-        #
-        # Read up on ssl_ciphers to ensure a secure configuration.
-        # See: https://bugs.debian.org/765782
-        #
-        # Self signed certs generated by the ssl-cert package
-        # Don't use them in a production server!
-        #
-        # include snippets/snakeoil.conf;
-
         root /var/www/wikijump/web/web;
 
         # Add index.php to the list if you are using PHP
         index index.php;
 
-        server_name wikijump;
+        location /wikijump {
+            # Assets
+            location /wikijump/assets {
+                alias /var/www/wikijump/web/storage/app/public;
+            }
 
-        rewrite ^/common--(.+)$ /files--common/$1 break;
+            # Fallback
+            try_files $uri /laravel.php?$args;
+        }
 
+        location / {
+            # First attempt to serve request as file, then
+            # as directory, then fall back to displaying a 404.
+            # As this is the most vague block, stuff in here gets examined last.
+            try_files $uri $uri/ /index.php?$args;
+            # Fallback Route
+        }
+
+        location ~ \.php$ {
+                    include snippets/fastcgi-php.conf;
+                    fastcgi_pass 127.0.0.1:9000;
+        }
+
+        rewrite ^/common--(.+)$ /files--common/$1 break;
```

Comment:
What is the origin of this rule? Aren't there kinds of `common--xxx` paths other than files?

Reply:
They're all in the `web/files--common` folder, under various subfolders. This is not modified from the original, only moved around.